eric_sse/entities.py

```
@@ -166,21 +164,29 @@ def deliver_next(self, listener_id: str) -> MessageContract:
         listener = self.get_listener(listener_id)
         if listener.is_running():
-            queue = self.__connection_manager.get_queue(listener.id)
-            msg = queue.pop()
-            listener.on_message(msg)
+            msg = self._get_connection(listener.id).fetch_message()
+            try:
+                listener.on_message(msg)
+            except Exception as e:
+                for handler in self.__listeners_error_handlers:
+                    handler.handle_on_message_error(msg=msg, exception=e)
+                    raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)